sips/pending/_posts/2017-02-07-make-types-behave-like-expressions.md

```diff
@@ -0,0 +1,187 @@
+---
+layout: sip
+disqus: true
+title: SIP-NN - Make types behave like expressions 
```

Comment:
I agree with @DarkDimius that a better name is desirable -- this one makes you wonder: "Behave like expression in regard to what?". I think Dmitry's suggestion is good, I would just add 'Make' at the beginning of the sentence: `Make types have precedence and associativity rules as expressions`. I'm not sure whether 'like' would be more appropriate in that case, maybe you can choose @soronpo, my non-native English is not that good.